ntoskrnl/kdbg/kdb_symbols.c

```diff
@@ -404,14 +405,15 @@ KdbpSymLoadModuleSymbols(
 
     DPRINT("Loading symbols from %wZ...\n", FileName);
 
-    if (!RosSymCreateFromFile(&FileHandle, RosSymInfo))
+    Result = RosSymCreateFromFile(&FileHandle, RosSymInfo);
```

Comment:
A possible alternative to save from the usage of this new variable, would be to do something like:
```c
    Status = (RosSymCreateFromFile(&FileHandle, RosSymInfo) ? STATUS_SUCCESS : STATUS_UNSUCCESSFUL);
```
and using `if (!NT_SUCCESS(Status))` (or `if (Status != STATUS_SUCCESS)`.

Reply:
I personally think this is really bad coding style, it obscures the true function of the code. I don't understand why such contortions would be considered more acceptable than an extra variable.

Reply:
i agree.